HappyAnding/HappyAnding/Views/Components/AdminCurationCell.swift

```diff
@@ -7,14 +7,74 @@
 
 import SwiftUI
 
+/*
+ 어드민 큐레이션 썸네일입니다.
+ 
+ #parameters
+ - curationThumbnail: 썸네일 배경의 이미지명
+ - title: 큐레이션 제목
+ - subtitle: 큐레이션 부제목
+ 
+ #description
+ - 어드민 큐레이션 데이터가 만들어진 후 데이터 연결이 필요합니다.
+ - padding 값이 설정되어있지 않습니다. 사용 시 padding 설정 후 사용해주세요.
+ */
+
 struct AdminCurationCell: View {
+    
+    let curationThumbnail: String
+    let title: String
+    let subtitle: String
+    
+    let cornerRadius: CGFloat = 12
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        ZStack(alignment: .top) {
+            NavigationLink(destination: ReadCurationView()) {
+                EmptyView()
+            }
+            titleAndSubtitle
+                .padding([.leading, .bottom, .trailing], 24.0)
```

Comment:
```suggestion
                .padding([.horizontal, .bottom], 24)
```

leading + trailing => horizontal
.top + .bottom => vertical로 표현할 수 있어요!

Reply:
앗 제가 테스트 후에 수정을 안 해놨군요... 수정하겠습니다